website/client/src/components/notifications.vue

```diff
@@ -140,82 +137,43 @@ import streak from './achievements/streak';
 import ultimateGear from './achievements/ultimateGear';
 import wonChallenge from './achievements/wonChallenge';
 import genericAchievement from './achievements/genericAchievement';
-import justAddWater from './achievements/justAddWater';
-import lostMasterclasser from './achievements/lostMasterclasser';
-import mindOverMatter from './achievements/mindOverMatter';
 import loginIncentives from './achievements/login-incentives';
 import onboardingComplete from './achievements/onboardingComplete';
 import verifyUsername from './settings/verifyUsername';
 import firstDrops from './achievements/firstDrops';
 
 const NOTIFICATIONS = {
-  CHALLENGE_JOINED_ACHIEVEMENT: {
-    achievement: true,
-    label: $t => `${$t('achievement')}: ${$t('joinedChallenge')}`,
-    modalId: 'joined-challenge',
-  },
-  ULTIMATE_GEAR_ACHIEVEMENT: {
-    achievement: true,
-    label: $t => `${$t('achievement')}: ${$t('gearAchievementNotification')}`,
-    modalId: 'ultimate-gear',
-  },
-  GUILD_JOINED_ACHIEVEMENT: {
-    label: $t => `${$t('achievement')}: ${$t('joinedGuild')}`,
-    achievement: true,
-    modalId: 'joined-guild',
-  },
-  INVITED_FRIEND_ACHIEVEMENT: {
-    achievement: true,
-    label: $t => `${$t('achievement')}: ${$t('invitedFriend')}`,
-    modalId: 'invited-friend',
-  },
+  // general notifications
   NEW_CONTRIBUTOR_LEVEL: {
     achievement: true,
     label: $t => $t('modalContribAchievement'),
     modalId: 'contributor',
     sticky: true,
   },
-  ACHIEVEMENT_ALL_YOUR_BASE: {
-    achievement: true,
-    label: $t => `${$t('achievement')}: ${$t('achievementAllYourBase')}`,
-    modalId: 'generic-achievement',
-    data: {
-      achievement: 'allYourBase', // defined manually until the server sends all the necessary data
-    },
-  },
-  ACHIEVEMENT_BACK_TO_BASICS: {
+  // achievement notifications
+  ACHIEVEMENT: { // data filled in handleUserNotifications
```

Comment:
If we're starting to pass the necessary information through via `data` in the notification object, can we go all the way down to this as the notification type for all achievements? Do we still need to split it out into several at all?

Reply:
I didn't update these because they're related to the orientation/tutorial for new users, but certainly something I can work on! Do you want it as part of this PR or a future one?

Reply:
Do you mean collapsing all the achievement types down into one item called `ACHIEVEMENT`: instead of `ACHIEVEMENT_CHALLENGE_JOINED`, `ACHIEVEMENT PARTY_ON`, etc?